rosbag_snapshot/src/snapshotter.cpp

```diff
@@ -303,37 +300,9 @@ MessageQueue::range_t MessageQueue::rangeFromTimes(Time const& start, Time const
   return range_t(begin, end);
 }
 
-bool MessageQueue::isLatched()
+bool MessageQueue::is_latched()
```

Comment:
Why switch from camel case to snake case? [Other methods](https://github.com/locusrobotics/rosbag_snapshot/pull/3/files#diff-4287ce149c1252338c2dd82ad59f76a918329e43941d0e57463db306b5358569R164) here are camel cased, and that matches our standard.

Reply:
Getting mixed up. I forgot that functions are camelCase and variables are snake_case. I'm used to Python and Rust where I do snake_case for functions.